scripts/check-nfs-server-service.py

```diff
@@ -3,7 +3,10 @@
 # zenoss-inspector-deps nfs-server-service.sh
 # zenoss-inspector-info
 
+
 def main():
+    conf_file="/etc/systemd/system/nfs-server.service.d/nfs-server.conf"
```

Comment:
This same script needs to work on systems configured for 1.1.x using 1.1.x guidelines.
So if CC version is '1.0.x' or '1.1.x', then conf_file=/lib/systemd/system/nfs-server.service otherwise conf_file= /etc/systemd/system/nfs-server.service.d/nfs-server.conf